.github/workflows/check-spelling.yml

```
@@ -7,29 +7,33 @@
-          experimental_apply_changes_via_bot: 1
-          check_file_names: 1
+    spelling:
+        name: Spell checking
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+            - name: check-spelling
+              uses: check-spelling/check-spelling@068774e3766c629d0d46819a253cfffcfff05eb5
+
+              with:
+                  config: .github/actions/spelling
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 16 but found 18

(indentation)